docs/system-api.md

```diff
@@ -138,6 +138,11 @@ SystemJS registry object supporting:
 
 See also [SystemJS.newModule](#systemjsnewmoduleobject---module).
 
+#### SystemJS.resolveSync(moduleName)
```

Comment:
Optional second argument here too - `moduleName, [parentName])`

Perhaps something like synchronous alternative to `SystemJS.resolve`.